environment.yml

```diff
@@ -4,4 +4,9 @@ channels:
   - defaults
 dependencies:
   - python=3.7
-  - xlwt=1.3.0=py37_0
\ No newline at end of file
+  - xlwt=1.3.0=py37_0
+  - pip
```

Comment:
Is this is typo? -pip twice

Reply:
I believe you need it twice. Once for downloading pip itself and the second time for downloading via pip. https://stackoverflow.com/a/35245610